Business resume homepage/contact.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="homepage.css">
+    <link rel="stylesheet" href="contact.css">
+    <link rel="stylesheet" href="resume.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Business Resume</title>
+</head>
+<body>
+    <header>
+        <div class="header-left">
+            <div class="header-box">
+                <div class="square">
+                </div>
+                <a href="index.html">Maya Nelson</a>
+            </div>
+            <span>/</span>
+            <a class="title-header" href="index.html">PROJECT MANAGER</a>
+        </div>
+
+        <div class="header-right">
+            <input id="toggle" type="checkbox"></input>
+
+            <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+            </label>
+
+            <div class="nav">
+            <div class="nav-wrapper">
+            <nav>
+                <ul>
+                    <li><a href="index.html">About me</a></li>
+                    <li><a href="resume.html">Resume</a></li>
+                    <li><a href="#">Projects</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <main>
+        <div class="title-main">
+            <div class="square">
+                    </div>
+            <h1>Let's talk</h1>
+        </div>
+       <form>
+            <div class="names-form">
```

Comment:
Is there a semantic html element for a set of fields?

Reply:
if you mean fieldset there is a bug for that, flex-box doesn't work very well with that, that's why I rather use div :)

Reply:
Can you explain more about the bug you encountered using fieldset?

Reply:
I don't remember exactly but when I was trying to make center my input elements (flex-direction column), justify and align didn't work properly and dev tools showed me fieldset blocked them and I changed it to div then it worked perfectly :D

Reply:
It's usually a mistake to sacrifice meaning for style. By this I mean, try not to use a div just because you encounter a difficulty with the proper semantic class. When you're hitting a blocker like this, the answer is to dig in and try to understand what is going on.

I made a quick MRE of this code using a fieldset. Why don't you try forking this codepen and building up until you can recreate your bug? It should make it much clearer what is going on.

https://codepen.io/IntroToTech/pen/poKdOdm

Reply:
got your point, 
https://codepen.io/shayan-mahnam/pen/GRGOXeB
here everything works as I wanted.

so I added fieldset.


thank you so much for reviewing.